runtime/lua/vim/lsp/util.lua

```diff
@@ -213,9 +273,9 @@ end
 -- https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_completion
 local function get_completion_word(item)
   if item.textEdit ~= nil and item.textEdit.newText ~= nil then
-    return item.textEdit.newText
+    return M.parse_snippet(item.textEdit.newText)
   elseif item.insertText ~= nil then
-    return item.insertText
+    return M.parse_snippet(item.insertText)
```

Comment:
If `item.insertTextFormat ~= nil and protocol.InsertTextFormat[item.insertTextFormat] == "PlainText"`, we can skip parse snipet.

Reply:
done